build/ci/official.yml

```
@@ -61,117 +88,182 @@ steps:
     testRunTitle: 'Unit Test Results'
   condition: succeededOrFailed()
 
-- task: PowerShell@2
-  inputs:
-    filePath: build\variables\_vsts.ps1
-    failOnStderr: true
-  displayName: Set VSTS variables based on source 
-
-# Symbols is the one thing we cannot declare in the ps1 artifact scripts due to the extra steps
-# needed and the specific order it has to be performed in. Copy -> Publish Symbols -> Publish
-# Artifact -> publish to symweb
+###################################################################################################################################################################
+# PUBLISH SYMBOLS
+###################################################################################################################################################################
 
+# Symbols is the one thing we cannot declare in the ps1 artifact scripts due to the extra steps needed and the specific order it has to be performed in.
+# Copy -> Publish Symbols -> Publish Artifact -> Publish to SymWeb
 - task: CopyFiles@2
+  displayName: Copy Symbols
   inputs:
     SourceFolder: '$(Build.SourcesDirectory)\artifacts\$(BuildConfiguration)'
     Contents: |
       **/Microsoft.VisualStudio.ProjectSystem.Managed?(*.pdb|*.dll|*.xml)
       **/Microsoft.VisualStudio.AppDesigner?(*.pdb|*.dll|*.xml)
       **/Microsoft.VisualStudio.Editors?(*.pdb|*.dll|*.xml)
-    TargetFolder: $(Build.ArtifactStagingDirectory)/symbols
-  displayName: Copy symbols
+    TargetFolder: $(Build.ArtifactStagingDirectory)/Symbols
 
+# https://docs.microsoft.com/azure/devops/pipelines/tasks/build/index-sources-publish-symbols
 - task: PublishSymbols@2
-  displayName: Archive symbols to VSTS
+  displayName: Publish Symbols to Azure Artifacts
   inputs:
-    SymbolsFolder: $(Build.ArtifactStagingDirectory)/symbols
-    SearchPattern: 
-      '**/*.pdb
+    SymbolsFolder: '$(Build.ArtifactStagingDirectory)/Symbols'
+    SearchPattern: |
+      **/*.pdb
       **/*.dll
-      **/*.exe'
-    publishSymbols: true
+      **/*.exe
+    # NOTE: This produces a warning about indexing (if enabled) but will still publish the symbols. See: https://github.com/microsoft/azure-pipelines-tasks/issues/15605
+    IndexSources: false
+    PublishSymbols: true
     SymbolServerType: TeamServices
-    SymbolsProduct: 'Roslyn Project System'
+    SymbolsProduct: 'DotNet Project System'
```

Is this just a display string? If so the correct branding would be:

```suggestion
    SymbolsProduct: '.NET Project System'
```

---

I actually hunted for where this information was used and couldn't find an answer. However, it is the only place we still seem to use the term "Roslyn Project System" in the repo, so I figured updating it made sense. I changed it to use `.NET` now so we should be consistent with the rest of the repo/branding.